### DISCUSSION TITLE
Unable to run Video File

### DISCUSSION THREAD

**Discussion by l33t-c0d3r-66** [AbdulJabbar64/Live-Video-To-Sketch-OpenCV#6] [Q&A] [answered]
Hello,
I'm Unable to run the video file any ideas about this issue?

**Comment by AbdulJabbar64** [comment, marked as answer]
First you check your videocam port if you are use front videocam your use cv.VideoCapture(0) else use cv.VideoCapture(1) for back videocam

**Reply by l33t-c0d3r-66** [reply]
Thanks it worked :)
